.github/workflows/release.yaml

```diff
@@ -3,59 +3,67 @@ name: Release
 on:
   push:
     tags:
-      - "*"
+      - v[0-9]+.*
 jobs:
-  build:
-    strategy:
-      matrix:
-        os: [ubuntu-latest, macos-latest]
-        rust: [stable]
+  body:
+    runs-on: ubuntu-latest
     permissions:
       contents: write
-
-    runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout
-        uses: actions/checkout@master
-
-      - name: Install just
-        uses: taiki-e/install-action@just
-
-      - name: Install Rust
-        uses: actions-rs/toolchain@v1
+        uses: actions/checkout@v4
+      - name: Extract release notes
+        uses: ffurrer2/extract-release-notes@v2
+        id: release_notes
+      - name: Upload Body
+        uses: softprops/action-gh-release@v2
         with:
-          toolchain: stable
+          body: ${{ steps.release_notes.outputs.release_notes }}
 
-      - name: Setup build env for Linux
-        if: matrix.os == 'ubuntu-latest'
+  build:
+    permissions:
+      contents: write
+    continue-on-error: true
```

Comment:
I think this should be false. what do you think ?

Reply:
I put it to true, because when trying to compile for linux on arm all of them failed as the one compile failed.

Reply:
If you add some variants in the future, maybe for other os or arch, then you at least can rest assured that the previous compiles still do their job.

Reply:
I mean isn't that what should happen when you try to release a new version ? I do not want to have a  new release with some binaries missing from an arch cause it has failed to compile. in that case something is wrong in the code and it should be fixed before releasing.

Reply:
I know what you mean. But as an example:
* I have a cross platform rust program.
* I know this as every library advertises that it is this way.
* I know that it works for my own arch and os.
* I don't have any other arch or os.
* Tho as it compiles for those it probably works.
* I change something in my program.
* It still compiles for my own arch and os.
* I publish it on crates.io (no going back).
* When looking at github whups no builds all of them failed.
Or 
* When looking at github whups the build for risv64 failed. All others are there as expected.

Reply:
I mean before doing a release I myself don't compile for all arch and os before doing the release. Do you? You probably don't have a windows pc, so compiling it beforehand for every arch including windows will be quit annoying.

Reply:
moreover if one of them fails it is still displayed that one failed.
So you will still see that there is a problem.

Reply:
Why not accept it for now, and change it to false in the future, when you feel that it is a problem.

Reply:
I prefer to prevent issues than fixing them when they occur, and this is one of the issue we can prevent actually.
would be really nice if you can change it to `false` :pray:

Reply:
done